### ISSUE TITLE
Rules?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/ateam-master-reference#2]
@iteles is any of this: https://github.com/ideaq/rules  relevant...?

**Comment by iteles** [comment]
Yes, all of it in one area or another (client communication, internal processes, code of conduct)! I think one repo may not be enough to contain this. It'll get out of hand - needs to be more like a reference book.

So much to do, but **so** excited! :blush:

**Comment by Cleop** [comment]
@iteles - this issue seems to be quite vague. Perhaps you can help give it a little more direction by updating it with what is still relevant to be done? Or is it now totally out of date?
